fizzbuzz.py

```diff
@@ -0,0 +1,20 @@
+#function fizzbuzz
+def fizzbuzz(a, b):
+    if isinstance((a and b), list):
+        combined_length = len(a) + len(b)
+        
+        if ((len(a) + len(b)) % 3 == 0) and ((len(a) + len(b)) % 5 == 0):
```

Comment:
You already created a variable combined_length containing the length why do you have to repeat calculating it here?
It would be better if you just write 
```python
if(combined_length%3 == 0 and combined_length % 5 == 0 ):
      # do some thing
```

Reply:
Thanks @dennisja , I just for got to reuse it since it had failed to function on the two conditions when it was an elif. Let me rectify it